src/app/Fake.DotNet.MSBuild/MSBuild.fs

```diff
@@ -90,22 +89,23 @@ type MSBuildLoggerConfig = MSBuildDistributedLoggerConfig
 #if !NO_VSWHERE // legacy fakelib
 module private MSBuildExeFromVsWhere =
     open BlackFox.VsWhere
-    open System.Text.RegularExpressions
+    open System.Diagnostics
 
     let private getAllVsPath () =
-        VsInstances.getWithPackage "Microsoft.Component.MSBuild" false
+        VsInstances.getWithPackage "Microsoft.Component.MSBuild" true
```

Comment:
Not sure about defaulting to using prereleases now...

Reply:
Final release is in 2 weeks, I don’t object to reverting this to `false`.

Reply:
I've actually just checked this, and VS 2019 RC isn't actually marked as a pre-release. So it works with `false` anyway. I'll revert this change.

Reply:
Yes RC should be tagged as stable and the current preview (as they run 2 branches in parallel) as prerelease

Reply:
Makes sense. Have changed. Also added similar log message to VS2017 for consistency.